test/e2e_new/data/infrastructure-aws/cluster-template-multi-az.yaml

```diff
@@ -0,0 +1,125 @@
+---
+apiVersion: cluster.x-k8s.io/v1alpha3
+kind: Cluster
+metadata:
+  name: "${CLUSTER_NAME}"
+spec:
+  clusterNetwork:
+    pods:
+      cidrBlocks: ["192.168.0.0/16"]
+  infrastructureRef:
+    apiVersion: infrastructure.cluster.x-k8s.io/v1alpha3
+    kind: AWSCluster
+    name: "${CLUSTER_NAME}"
+  controlPlaneRef:
+    kind: KubeadmControlPlane
+    apiVersion: controlplane.cluster.x-k8s.io/v1alpha3
+    name: "${CLUSTER_NAME}-control-plane"
+---
+apiVersion: infrastructure.cluster.x-k8s.io/v1alpha3
+kind: AWSCluster
+metadata:
+  name: "${CLUSTER_NAME}"
+spec:
+  region: "${AWS_REGION}"
+  sshKeyName: "${AWS_SSH_KEY_NAME}"
+  networkSpec:
+    subnets:
+      - availabilityZone: "${AWS_AVAILABILITY_ZONE_1}"
```

Comment:
What was the reason to do it this way instead of
```
  networkSpec:
    vpc:
      availabilityZoneUsageLimit: 3
```
?

Reply:
The test that uses this template forces creating machine deployments in specific subnets. Without it, the instances would get created in the first available subnet and there's no validation on multi-az for worker nodes.

Reply:
From other conversations, this is needed for worker node distribution.